t/selenium/External_Links_Editor.json5

```diff
@@ -414,14 +414,9 @@
     },
     // duplicate the first URL again by editing the other existing URL
     {
-      command: 'sendKeys',
-      target: "xpath=(//table[@id='external-links-editor']//input[@type='url'])[1]",
-      value: '${KEY_HOME}${KEY_SHIFT}${KEY_END}${KEY_SHIFT}${KEY_BKSP}',
-    },
-    {
-      command: 'select',
-      target: "xpath=//table[@id='external-links-editor']//tr[2]//select[contains(@class, 'link-type')]",
-      value: 'label=regexp:\\B',
+      command: 'click',
+      target: "xpath=//table[@id='external-links-editor']//tr[2]//button[contains(@class, 'remove-item')]",
+      value: '',
```

Comment:
Does it still “duplicate the first URL”?

Reply:
I think so, please see the latest total diff. This comment is for the next action to edit an exising link to duplicate the deleted first URL.

Reply:
Oh ok, you restored it, thanks.